### ISSUE TITLE
2.120.5 - unused_variables - false positive no 1

### ISSUE THREAD

**Issue by Zeatynis** [abaplint/abaplint#4138]
<img width="662" height="231" alt="Image" src="https://github.com/user-attachments/assets/bc033f7e-cdab-4390-b7ab-a2153046bfa1" />

```
REPORT zfoo.

SELECT-OPTIONS s_bukrs FOR t001-bukrs.

AT SELECTION-SCREEN ON s_bukrs.
```
